packages/ember-testing/lib/setup_for_testing.js

```diff
@@ -35,11 +35,13 @@ export default function setupForTesting() {
     setAdapter((typeof self.QUnit === 'undefined') ? new Adapter() : new QUnitAdapter());
   }
 
-  jQuery(document).off('ajaxSend', incrementPendingRequests);
-  jQuery(document).off('ajaxComplete', decrementPendingRequests);
+  if (jQuery) {
```

Comment:
can you do `import { jQuery } from 'ember-views';` if there is no jquery? What does that import?